tensilelite/Tensile/LibraryLogic.py

```diff
@@ -503,15 +515,33 @@ def addFromCSV(self, dataFileName, numSolutions, solutionMap):
                 winnerGFlops = gflops
               solutionIdx += 1
 
+          # calculate effLike
+          # effLike = winnerGFlops / Frequency(MHz)
+          try:
+            frequency = float(row[columnOfFreqIdx])
+            if frequency != 0 and not math.isnan(frequency):
+              effLike = round(float(winnerGFlops) / frequency, 2)
+            else:
+              effLike = float(winnerGFlops)
+              print1("Warning: Frequency is Nan or 0, using GFlops as the efficiency metric")
+          except(ValueError, TypeError):
+            effLike = float(winnerGFlops)
+            print1("Warning: Could not convert frequency value to float, using GFlops as the efficiency metric")
```

Comment:
per our discussion, please let user know that the efflike performance can't be calculated and ask your for confirmation they still want the output by using gflops.

Reply:
Feature added. The process will wait for the input from the user when there is an error. The user could choose to abort the process or proceed with winnerGFlops as the final performance metric.
![image](https://github.com/user-attachments/assets/32dc83fa-71f2-44b0-a477-250af31aba38)

Please let me know if there is any further problems.